wiki/snapshot.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Snapshots
+layout: wiki
+---
+
+# Overview
+Snapshots are a snapshot of the chain at a particular blockheight.
```

Comment:
The sentence may be a bit technical for the first sentence. Perhaps something like this would be better
```suggestion
Snapshots are a copy of the blockchain at a particular point in time.
```